wiki/Beatmap/Packs/en.md

```diff
@@ -40,8 +40,6 @@ The beatmaps packs can be found and downloaded from: [osu! Beatmaps Packs page](
 ### Torrenting
 
 - [osu! Torrent Beatmap Packs](https://osu.ppy.sh/community/forums/topics/687910)
-  - [PandoTracker](https://pandotracker.me) by [- Pandoro](https://osu.ppy.sh/users/2574057)
-- [Beatmap-Torrents](https://osu.ppy.sh/community/forums/topics/147478) (closed)
-  - [HiraOsu! - Torrents](http://osu.hiramiya.me/torrents.htm) by [Hiramiya](https://osu.ppy.sh/users/1313441)
+  - [PandoTracker](https://pandotracker.me "PandoTracker") by [- Pandoro](https://osu.ppy.sh/users/2574057)
```

Comment:
Isn't this kinda thing a step backwards? I don't think we ever want to be showing tooltips with the same content as the link itself..